neural_compressor/adaptor/pytorch.py

```diff
@@ -3148,7 +3150,7 @@ def _pre_hook_for_qat(self, dataloader=None):
 
         # PyTorch 1.13 and above version, need example_inputs for fx trace, but it not realy used,
         # so set it to None.
-        example_inputs = None
+        self.example_inputs = None
```

Comment:
remove it?

Reply:
For fine-grianed fx, the example_inputs for each module maybe a big issue, so I keep it to notice us that example_inputs is not used right now.